pandas/io/json/_json.py

```diff
@@ -737,17 +722,11 @@ def read(self):
         """
         Read the whole JSON input into a pandas object.
         """
-        if self.lines:
-            if self.chunksize:
-                obj = concat(self)
-            elif self.nrows:
-                lines = list(islice(self.data, self.nrows))
-                lines_json = self._combine_lines(lines)
-                obj = self._get_object_parser(lines_json)
-            else:
-                data = ensure_str(self.data)
-                data = data.split("\n")
-                obj = self._get_object_parser(self._combine_lines(data))
+        if self.lines and self.chunksize:
```

Comment:
I think you might have an issue merging master - none of this should have changed

Reply:
Yeah. Fixed now